Detections/SecurityEvent/UserCreatedAddedToBuiltinAdmins_1d.yaml

```diff
@@ -74,25 +74,32 @@ query: |
   //Create User first, then the add to the group.
   | project Computer, CreatedUserTime, CreatedUserEventID, CreatedUserActivity, CreatedUser, CreatedUserSid, GroupAddTime, GroupAddEventID, 
   GroupAddActivity, AccountUsedToCreateUser, GroupName, GroupSid, AccountThatAddedUser, SIDofAccountThatAddedUser 
-  | extend timestamp = CreatedUserTime, AccountCustomEntity = CreatedUser, HostCustomEntity = Computer
+  | extend timestamp = CreatedUserTime
+  | extend HostName = tostring(split(Computer, ".")[0]), DomainIndex = toint(indexof(Computer, '.'))
+  | extend HostNameDomain = iff(DomainIndex != -1, substring(Computer, DomainIndex + 1), Computer)
+  | extend AccountName = tostring(split(CreatedUser, "@")[0]), AccountUPNSuffix = tostring(split(CreatedUser, "@")[1])
 entityMappings:
   - entityType: Account
     fieldMappings:
-      - identifier: FullName
-        columnName: AccountCustomEntity
+      - identifier: Name
+        columnName: AccountName
```

Comment:
CreatedUser and AccountThatAddedUser are not UPNs in this case. They are Name + NTDomain. 

CreatedUser and AccountThatAddedUser are actually built from separated entity types already. This is a legacy method.  You can get what you need by the following:
| extend TargetUserName = tostring(EventData.TargetUserName), TargetDomainName = tostring(EventData.TargetDomainName)

Then entity map these to Name and NTDomain.

Also, please map another entity to Account for AccountThatAddedUser which will be SubjectUserName and SubjectDomainName.